src/naive/datetime/tests.rs

```diff
@@ -338,6 +338,40 @@ fn test_datetime_parse_from_str() {
     );
 }
 
+#[test]
```

Comment:
Please split this into two commits. As a reminder, moving code and changing it in the same code makes it harder to review the changes.

Reply:
Yes, of course.